src/engine/sync/basesyncablelistener.h

```diff
@@ -55,8 +55,7 @@ class BaseSyncableListener : public SyncableListener {
     // pSource.
     void setMasterBeatDistance(Syncable* pSource, double beat_distance);
 
-    void setMasterParams(Syncable* pSource, double beat_distance,
-                         double base_bpm, double bpm);
+    void setMasterParams(Syncable* pSource, double beat_distance, double base_bpm, double bpm);
```

Comment:
Unrelated to this PR, but long term we should change it to:

```suggestion
    void setLeaderParams(Syncable* pSource, double beatDistance, mixxx::Bpm baseBpm, mixxx::Bpm bpm);
```

Reply:
a refactor PR is a good time to fix these

Reply:
mmm I spoke too soon, the Bpm class needs a bunch of overrides to make it more usable:
```
/home/owen/src/github/mixxx/src/engine/sync/basesyncablelistener.cpp: In constructor ‘BaseSyncableListener::BaseSyncableListener(UserSettingsPointer)’:
/home/owen/src/github/mixxx/src/engine/sync/basesyncablelistener.cpp:18:36: error: cannot convert ‘double’ to ‘mixxx::Bpm’
   18 |     m_pInternalClock->setMasterBpm(124.0);
      |                                    ^~~~~
      |                                    |
      |                                    double
```

also operator== with literals, etc.